rfcs/text/0000-rfc-process.md

```diff
@@ -0,0 +1,174 @@
+### DCP PR:
+
+# RFC Process
+
+## Summary
+
+The Human Cell Atlas Data Coordination Platform (HCA DCP) evolved informal, consensus-driven processes to coordinate and document technical decisions.
```

Comment:
If there are also governance processes decided this probably needs to be changeed to "document technical and governance decisions"

Reply:
WFM